docs/src/refs.bib

```diff
@@ -1,4 +1,158 @@
 
+@Article{BS84,
+  author        = {Beynon, W. M. and Spaltenstein, N.},
+  title         = {Green functions of finite {Chevalley} groups of type {{\(E_ n (n=6,7,8)\)}}},
+  zbl           = {0539.20025},
+  journal       = {J. Algebra},
+  fjournal      = {Journal of Algebra},
+  volume        = {88},
+  pages         = {584--614},
+  year          = {1984},
+  doi           = {10.1016/0021-8693(84)90084-X},
+  language      = {English},
+  zbmath        = {3857372}
+}
+
+@InCollection{CR74,
+  author        = {Chang, Bomshik and Ree, Rimhak},
+  title         = {The characters of {{\(G_ 2(q)\)}}},
```

Comment:
I think this won't render right, you need to use `$` for math mode (full LaTeX is not supported). 
```suggestion
  title         = {The characters of $G_2(q)$},
```